bolides.js

```diff
@@ -577,11 +626,35 @@ var bolides = {
         bolides.canvas.ctx.drawImage(bolides.images.bullet, -3, -12.5);
         bolides.canvas.ctx.restore();
         // Draw the 1st asteroid
-        bolides.canvas.ctx.drawImage(bolides.images.asteroid, bolides.asteroid1.x, bolides.asteroid1.y, 62, 62);
+        if (bolides.asteroid1.isBolide) {
+          bolides.canvas.ctx.save();
+          bolides.canvas.ctx.translate(bolides.asteroid1.x + 31, bolides.asteroid1.y + 31);
+          bolides.canvas.ctx.rotate(bolides.asteroid1.angle);
+          bolides.canvas.ctx.drawImage(bolides.images.bolide, -31, -31, 62, 154);
+          bolides.canvas.ctx.restore();
+        } else {
+          bolides.canvas.ctx.drawImage(bolides.images.asteroid, bolides.asteroid1.x, bolides.asteroid1.y, 62, 62);
+        }
         // Draw the 2nd asteroid
-        bolides.canvas.ctx.drawImage(bolides.images.asteroid, bolides.asteroid2.x, bolides.asteroid2.y, 62, 62);
+        if (bolides.asteroid2.isBolide) {
+          bolides.canvas.ctx.save();
+          bolides.canvas.ctx.translate(bolides.asteroid2.x + 31, bolides.asteroid2.y + 31);
+          bolides.canvas.ctx.rotate(bolides.asteroid2.angle);
+          bolides.canvas.ctx.drawImage(bolides.images.bolide, -31, -31, 62, 154);
+          bolides.canvas.ctx.restore();
+        } else {
+          bolides.canvas.ctx.drawImage(bolides.images.asteroid, bolides.asteroid2.x, bolides.asteroid2.y, 62, 62);
+        }
         // Draw the 3rd asteroid
-        bolides.canvas.ctx.drawImage(bolides.images.asteroid, bolides.asteroid3.x, bolides.asteroid3.y, 62, 62);
+        if (bolides.asteroid3.isBolide) {
+          bolides.canvas.ctx.save();
+          bolides.canvas.ctx.translate(bolides.asteroid3.x + 31, bolides.asteroid3.y + 31);
+          bolides.canvas.ctx.rotate(bolides.asteroid3.angle);
+          bolides.canvas.ctx.drawImage(bolides.images.bolide, -31, -31, 62, 154);
+          bolides.canvas.ctx.restore();
```

Comment:
Rather than manually calling the same logic for each asteroid, you could store them in an array, then iterate over that:

```js
bolides.asteroids = [asteroid1, asteroid2, asteroid3]

bolides.asteroids.forEach(function(asteroid){
  bolides.canvas.ctx.save();
   bolides.canvas.ctx.translate(asteroid.x + 31, asteroid.y + 31);
   bolides.canvas.ctx.rotate(asteroid.angle);
    //etc... 
})